### ISSUE TITLE
Change alpha-value-notation for opacity property

### ISSUE THREAD

**Issue by GrimLink** [stylelint/stylelint-config-standard#210]
### What steps are needed to reproduce the bug?

Use stylelint version 14 with stylelint-config-standard version 23.

### What did you expect to happen?

No error on `opacity: 0.8;`

### What actually happened?

It throws an error on opacities with a number value, while this not supported in every browser yet -> https://developer.mozilla.org/en-US/docs/Web/CSS/opacity

```bash
_tests_/index.scss
 59:14  ✖  Expected "0.8" to be "80%"  alpha-value-notation
```

**Comment by GrimLink** [comment]
This rule should not be part of this config or should be exclude the css property `opacity`

**Comment by jeddy3** [comment]
@GrimLink Thanks for the report.

Let's exclude the `opacity` property until it lands in the last two version of Safari.

I've labelled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

**Comment by GrimLink** [comment]
I have time, and the PR is created 😉
